raiden-dapp/src/components/transaction-history/Transaction.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="transaction">
+    <v-row class="transaction__item" no-gutters>
+      <v-col class="transaction__item__icon" cols="1">
+        <v-img
+          v-if="transfer.direction === 'sent'"
+          height="45px"
+          width="45px"
+          :src="require('../../assets/sent_transfer.svg')"
+        />
+        <v-img
+          v-else
+          height="45px"
+          width="45px"
+          :src="require('../../assets/received_transfer.svg')"
+        />
+      </v-col>
+      <v-col class="transaction__item__details-left">
+        <v-row no-gutters>
+          {{
+            transfer.direction === 'sent'
+              ? $t('transfer-history.sent-title')
+              : $t('transfer-history.received-title')
+          }}
+          <address-display
+            class="transaction__item__details-left__address"
+            :address="transfer.partner"
+          />
+        </v-row>
+        <v-row class="transaction__item__details-left__time-stamp" no-gutters>
+          {{ new Intl.DateTimeFormat('en-US').format(transfer.changedAt) }}
+          {{ transfer.changedAt.toLocaleTimeString('en-US') }}
+        </v-row>
+      </v-col>
+      <v-col class="transaction__item__details-right">
+        <v-row no-gutters>
+          <div class="transaction__item__details-right__amount">
+            <span class="transaction__item__details-right__amount--direction">
+              {{
+                transfer.direction === 'sent'
+                  ? $t('transfer-history.sent-symbol')
+                  : $t('transfer-history.received-symbol')
+              }}
+            </span>
+            <amount-display
+              class="transaction__item__details-right__amount--sum"
+              exact-amount
+              :amount="transfer.amount"
+              :token="transfer.token"
+            />
+          </div>
+        </v-row>
+        <v-row no-gutters>
+          <span class="transaction__item__details-right--status">
+            <v-chip
+              v-if="transfer.success === undefined"
+              x-small
+              color="rgba(253, 211, 39, 0.3)"
```

Comment:
can we extract these colors to the theme.
You can apparently add custom colors in the theme https://github.com/raiden-network/light-client/blob/35309fcd3587e33b0aa31efc8e59bfa927ee96ee/raiden-dapp/src/plugins/vuetify.ts#L17 and then reference them by name. It would be better than harding colors in the template.